google-cloud-pub-sub/src/main/scala/akka/stream/alpakka/googlecloud/pubsub/Model.scala

```diff
@@ -7,7 +7,7 @@ package akka.stream.alpakka.googlecloud.pubsub
 import scala.collection.immutable
 import scala.collection.JavaConverters._
 
-final case class PubSubMessage(data: String, messageId: String)
+final case class PubSubMessage(data: String, messageId: String, publishTime: Option[String])
```

Comment:
[Instant](https://docs.oracle.com/javase/8/docs/api/java/time/Instant.html] instead of a string me thinks...